### PR TITLE
Make plaid-python quickstart support link token. Also use new shared

### PR DESCRIPTION
**Pull request by davidzhanghp**
front end code.

### CODE REVIEW

File: python/server.py [hunk 1]
```diff
@@ -24,67 +23,134 @@
...
+  global access_token
+  global item_id
+  return jsonify({
+    'item_id': item_id,
+    'access_token': access_token,
+    'products': PLAID_PRODUCTS
+  })
+
+@app.route('/api/create_link_token_for_payment', methods=['POST'])
+def create_link_token():
```
Comment (otherchen) [inline python/server.py:80]: I think there's a mix up here - should be `create_link_token_for_payment`

File: python/server.py [hunk 2]
```diff
@@ -24,67 +23,134 @@
...
+        }
+      }
+    )
+    pretty_print_response(response)
+    return jsonify(response)
+  except plaid.errors.PlaidError as e:
+    return jsonify(format_error(e))
+
+@app.route('/api/create_link_token', methods=['POST'])
+def create_link_token_for_payment():
```
Comment (otherchen) [inline python/server.py:127]: should be `create_link_token`

File: python/server.py [hunk 3]
```diff
@@ -24,67 +23,134 @@
...
 @app.route('/oauth-response.html')
 def oauth_response():
   return render_template(
-    'oauth-response.ejs',
-    plaid_public_key=PLAID_PUBLIC_KEY,
-    plaid_environment=PLAID_ENV,
-    plaid_products=PLAID_PRODUCTS,
-    plaid_country_codes=PLAID_COUNTRY_CODES,
-    plaid_oauth_nonce=PLAID_OAUTH_NONCE,
+    'oauth-response.html',
```
Comment (otherchen) [inline python/server.py:56]: Just wondering if we've tested out that the oauth flow works here

File: python/README.md [hunk 4]
```diff
@@ -62,9 +70,8 @@ PLAID_PUBLIC_KEY='PUBLIC_KEY' \
 PLAID_ENV='sandbox' \
 PLAID_PRODUCTS='transactions' \
 PLAID_COUNTRY_CODES='GB,FR,ES,IE,NL' \
-PLAID_OAUTH_REDIRECT_URI='http://localhost:5000/oauth-response.html' \
-PLAID_OAUTH_NONCE='nice-and-long-nonce' \
+PLAID_OAUTH_REDIRECT_URI='http://localhost:8000/oauth-response.html' \
```
Comment (otherchen) [inline python/README.md:73]: rename to `PLAID_REDIRECT_URI`

File: python/server.py (hunk 3 above)
Comment (davidzhanghp) [reply python/server.py:56]: yes

**Pull request merged**